packages/elementor-ui/components/utils.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param styles - { base: { shared: '', variant: { h1: '', h2: '' } }, dark: { shared: '', variant: { h1: '', h2: '' } } }
```

Comment:
Can we make the shared CSS to be rendered only once per component type?

Reply:
Done (entire new solution for caching components).